src/microbe_stage/OrganelleLayout.cs

```diff
@@ -72,25 +72,30 @@ public bool CanPlace(T organelle, bool allowCytoplasmOverlap = false)
 
     /// <summary>
     ///   Returns true if CanPlace would return true and an existing
-    ///   hex touches one of the new hexes.
+    ///   hex touches one of the new hexes, or is the last hex and replaced.
     /// </summary>
-    public bool CanPlaceAndIsTouching(T organelle, bool allowCytoplasmOverlap = false)
+    public bool CanPlaceAndIsTouchingOrReplacingLast(T organelle, bool allowCytoplasmOverlap = false)
```

Comment:
Rather than change the name, add a new parameter for allowingReplacingLastCytoplasm that defaults to false.

Reply:
Added in f3ad887